test/clj/game/cards/agendas_test.clj

```diff
@@ -745,8 +745,7 @@
   ;; Dedicated Neural Net
   (testing "Corp chooses card to access. Issue #4874"
     (do-game
-      (new-game {:corp {:deck [(qty "Hedge Fund" 5)]
```

Comment:
One of the reasons I include these is to keep the game from being considered "won" during the course of normal games. I know we `clear-win` during set up, but I generally think it's better to not have to worry about that.